build/Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM alpine:3.8
+FROM openshift/origin-release:golang-1.10 AS build
+ENV OPERATOR_PATH=/go/src/github.com/openshift/dedicated-admin-operator
+COPY . ${OPERATOR_PATH}
+WORKDIR ${OPERATOR_PATH}
+RUN make build
 
-RUN apk upgrade --update --no-cache
-
-USER nobody
-
-ADD build/_output/bin/dedicated-admin-operator /usr/local/bin/dedicated-admin-operator
+FROM alpine:latest  
```

Comment:
Not pinning to a version is a risk in my mind, that we don't know what we're going to get.  Prefer go back to a specific tag, you had `alpine:3.8` before.  While on that, any reason why we're staying on `alpine` and not looking for a RH, openshift, or origin base image?

Reply:
Agreed, we want to pin to a specific version, whether it's alpine or something else.

Reply:
Ok, I'll lock on 3.8. It's alpine mostly cause the template generated by operator-sdk comes with that.

Another option would be having `FROM scratch`, thoughts?

Reply:
@rogbas about `scratch`, be sure to test it because it's possible that you don't have CA cert bundle, or `/etc/passwd` or other bits we take for granted which are relied upon by syscalls or libraries.

Reply:
That's true @lisa, just locked the alpine version for now.